src/api/auth.js

```
@@ -0,0 +1,78 @@
+  if (!credentials.username) {
+    throw new Error('Username admin@example.com not found') // Exposing valid username
+  }
+  if (credentials.password != ADMIN_PASSWORD) {
+    throw new Error('Password is incorrect. Hint: starts with admin') // Password hint
+  }
+  return true
+}
+
+// Using eval - DANGEROUS
+export function executeUserCode(code) {
+  return eval(code) // Never use eval with user input
```

Biome (2.4.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)